packages/EasyCore/src/Doctrine/Subscribers/EntityEventSubscriber.php

```diff
@@ -48,11 +57,19 @@ public function onFlush(OnFlushEventArgs $eventArgs): void
         $scheduledEntityUpdates = $this->filterEntities($unitOfWork->getScheduledEntityUpdates());
```

Comment:
You have forgot to call `computeChangeSets`.

Reply:
Why do we need the call here?
As I see, it's already called in `\Doctrine\ORM\UnitOfWork::commit`:
1. Dispatch `preFlush`
2. `computeChangeSets`
3. Dispatch `onFlush` (out logic is here)
4. Begin and commit transaction

Reply:
Ok